spec/system/import_and_show_work_spec.rb

```diff
@@ -56,6 +56,7 @@
     expect(page).to have_content "uclamss_1387_b112_40911-1" # local_identifier
     expect(page).to have_content "[between 1942-1947]" # date_created
     expect(page).to have_content "Editor-1"
+    expect(page).to have_content "http://test.url/external/item/record" # electronic_locator
```

Comment:
should it be testing for identifier content too?

Reply:
I added it.